### PR TITLE
BGP Service script path and error fix

### PR DESCRIPTION
**Pull request by vaibhavhd**
**- Why I did it**
Systemctl does not point to the new BGP service script, and fix errors in BGP script.

**- How I did it**
Updated new path and permission.

**- How to verify it**
Start/stop bgp service

Fixed the following error with existing bgp script `FAST_BOOT: command not found`:
```
admin@str-s6100-acs-4:~$ sudo systemctl stop bgp
admin@str-s6100-acs-4:~$ 
admin@str-s6100-acs-4:~$ sudo systemctl status bgp
â— bgp.service - BGP container
   Loaded: loaded (/lib/systemd/system/bgp.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Fri 2020-08-14 03:04:56 UTC; 3s ago
  Process: 17181 ExecStart=/usr/local/bin/bgp.sh wait (code=exited, status=0/SUCCESS)
  Process: 22970 ExecStop=/usr/local/bin/bgp.sh stop (code=exited, status=0/SUCCESS)
 Main PID: 17181 (code=exited, status=0/SUCCESS)

Aug 14 02:57:51 str-s6100-acs-4 bgp.sh[16941]: /usr/local/bin/bgp.sh: line 36: FAST_BOOT: command not found
Aug 14 02:57:53 str-s6100-acs-4 bgp.sh[16941]: Starting existing bgp container with HWSKU Force10-S6100
Aug 14 02:57:54 str-s6100-acs-4 bgp.sh[16941]: bgp
Aug 14 02:57:54 str-s6100-acs-4 systemd[1]: Started BGP container.
Aug 14 03:04:52 str-s6100-acs-4 systemd[1]: Stopping BGP container...
Aug 14 03:04:53 str-s6100-acs-4 bgp.sh[22970]: /usr/local/bin/bgp.sh: line 36: FAST_BOOT: command not found
Aug 14 03:04:56 str-s6100-acs-4 bgp.sh[22970]: bgp
Aug 14 03:04:56 str-s6100-acs-4 bgp.sh[17181]: 0
Aug 14 03:04:56 str-s6100-acs-4 systemd[1]: bgp.service: Succeeded.
Aug 14 03:04:56 str-s6100-acs-4 systemd[1]: Stopped BGP container.
admin@str-s6100-acs-4:~$ 
```

Fixed service start/stop:

Stop BGP:
```
admin@str-s6100-acs-4:~$ sudo systemctl stop bgp
admin@str-s6100-acs-4:~$ sudo systemctl status bgp
â— bgp.service - BGP container
   Loaded: loaded (/lib/systemd/system/bgp.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Fri 2020-08-14 05:58:33 UTC; 35s ago
  Process: 2032 ExecStart=/usr/local/bin/bgp.sh wait (code=exited, status=0/SUCCESS)
  Process: 6815 ExecStop=/usr/local/bin/bgp.sh stop (code=exited, status=0/SUCCESS)
 Main PID: 2032 (code=exited, status=0/SUCCESS)

Aug 14 05:52:33 sonic systemd[1]: Starting BGP container...
Aug 14 05:52:36 str-s6100-acs-4 bgp.sh[1434]: Creating new bgp container with HWSKU Force10-S6100
Aug 14 05:52:37 str-s6100-acs-4 bgp.sh[1434]: 7dcd03bb2710bc1b5bcfbed7aa8f93eacb55b69ac985a841b9a56fb34ca1db6d
Aug 14 05:52:40 str-s6100-acs-4 bgp.sh[1434]: bgp
Aug 14 05:52:40 str-s6100-acs-4 systemd[1]: Started BGP container.
Aug 14 05:58:30 str-s6100-acs-4 systemd[1]: Stopping BGP container...
Aug 14 05:58:33 str-s6100-acs-4 bgp.sh[2032]: 0
Aug 14 05:58:33 str-s6100-acs-4 bgp.sh[6815]: bgp
Aug 14 05:58:33 str-s6100-acs-4 systemd[1]: bgp.service: Succeeded.
Aug 14 05:58:33 str-s6100-acs-4 systemd[1]: Stopped BGP container.
admin@str-s6100-acs-4:~$ 
```
Start bgp:
```
admin@str-s6100-acs-4:~$ sudo systemctl start bgp
admin@str-s6100-acs-4:~$ sudo systemctl status bgp
â— bgp.service - BGP container
   Loaded: loaded (/lib/systemd/system/bgp.service; enabled; vendor preset: enabled)
   Active: active (running) since Fri 2020-08-14 05:59:19 UTC; 3s ago
  Process: 6880 ExecStartPre=/usr/local/bin/bgp.sh start (code=exited, status=0/SUCCESS)
 Main PID: 6966 (bgp.sh)
    Tasks: 11 (limit: 4915)
   Memory: 24.1M
   CGroup: /system.slice/bgp.service
           â”œâ”€6966 /bin/bash /usr/local/bin/bgp.sh wait
           â”œâ”€6967 /bin/bash /usr/bin/bgp.sh wait
           â””â”€6968 docker wait bgp

Aug 14 05:59:16 str-s6100-acs-4 systemd[1]: Starting BGP container...
Aug 14 05:59:16 str-s6100-acs-4 admin[6881]: Starting bgp service...
Aug 14 05:59:18 str-s6100-acs-4 bgp.sh[6880]: Starting existing bgp container with HWSKU Force10-S6100
Aug 14 05:59:19 str-s6100-acs-4 bgp.sh[6880]: bgp
Aug 14 05:59:19 str-s6100-acs-4 admin[6963]: Started bgp service...
Aug 14 05:59:19 str-s6100-acs-4 systemd[1]: Started BGP container.
admin@str-s6100-acs-4:~$ 
```
**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/build_templates/radv.service.j2 [hunk 1]
```diff
@@ -7,7 +7,7 @@ StartLimitIntervalSec=1200
 StartLimitBurst=3
 
 [Service]
-User={{ sonicadmin_user }}
+User=root
```
Comment (yxieca) [inline files/build_templates/radv.service.j2:10]: I am not sure this is a good change. @jleveque are you familiar with this part? The file above is also setting user to root directly.

Comment (jleveque) [reply files/build_templates/radv.service.j2:10]: Good catch. There is no `admin` user inside our containers, only `root`. This will not work. What is the reason for attempting to change the user here?

File: files/build_templates/per_namespace/bgp.service.j2 [hunk 2]
```diff
@@ -9,10 +9,10 @@ StartLimitIntervalSec=1200
...
 
 [Service]
-User={{ sonicadmin_user }}
-ExecStartPre=/usr/bin/{{docker_container_name}}.sh start{% if multi_instance == 'true' %} %i{% endif %}
-ExecStart=/usr/bin/{{docker_container_name}}.sh wait{% if multi_instance == 'true' %} %i{% endif %}
-ExecStop=/usr/bin/{{docker_container_name}}.sh stop{% if multi_instance == 'true' %} %i{% endif %}
+User=root
+ExecStartPre=/usr/local/bin/{{docker_container_name}}.sh start{% if multi_instance == 'true' %} %i{% endif %}
+ExecStart=/usr/local/bin/{{docker_container_name}}.sh wait{% if multi_instance == 'true' %} %i{% endif %}
+ExecStop=/usr/local/bin/{{docker_container_name}}.sh stop{% if multi_instance == 'true' %} %i{% endif %}
```
Comment (yxieca) [inline files/build_templates/per_namespace/bgp.service.j2:15]: Please split this BGP change into a different PR. And please remove the user change.

File: files/build_templates/radv.service.j2 (hunk 1 above)
Comment (jleveque) [reply files/build_templates/radv.service.j2:10]: As discussed offline, it should be OK.

Comment (yxieca) [reply files/build_templates/radv.service.j2:10]: Please remove this change above.

File: files/build_templates/per_namespace/bgp.service.j2 (hunk 2 above)
Comment (vaibhavhd) [reply files/build_templates/per_namespace/bgp.service.j2:15]: Changes are separated now.

File: files/build_templates/radv.service.j2 (hunk 1 above)
Comment (vaibhavhd) [reply files/build_templates/radv.service.j2]: Fixed. Thanks!

**Comment by vaibhavhd** [comment]
retest this please

**Comment by jleveque** [comment]
Retest this please

**Comment by vaibhavhd** [comment]
> @vaibhavhd: Do you still need to give radv.sh executable permission? It looks like that change was reverted as well.

Hi @jleveque, we still need that change. But, I have _segregated_ that change to a new small PR. - https://github.com/Azure/sonic-buildimage/pull/5186

**Comment by vaibhavhd** [comment]
Retest this please

**Comment by vaibhavhd** [comment]
Retest this please
